### ISSUE TITLE
alignment error

### ISSUE THREAD

**Issue by CoinCheung** [astanin/python-tabulate#70]
Hi, 

My code is like this: 
```python
from tabulate import tabulate


res = [{'object': '1986', 'relation': '出生日期', 'subject': '爱德华·尼科·埃尔南迪斯'}, {'object': '哥伦比亚', 'relation': '国籍', 'subject': '爱德华·尼科·埃尔南迪斯'}, {'object': '70公分', 'relation': '身高', 'subject': '爱德华·尼科·埃尔南迪斯'}]

title = ['subject', 'relation', 'object']
tbl = [title] + [[el['subject'], el['relation'], el['object']] for el in res]
tbl = tabulate(tbl, headers='firstrow', tablefmt='orgtbl', colalign=('right',))

print(tbl)
```

And the result is: 
```
|      subject | relation   | object   |
|--------------+------------+----------|
| 爱德华·尼科·埃尔南迪斯 | 出生日期       | 1986     |
| 爱德华·尼科·埃尔南迪斯 | 国籍         | 哥伦比亚     |
| 爱德华·尼科·埃尔南迪斯 | 身高         | 70公分     |
```
How could I make the column aligns well ?

**Comment by themrdjj** [comment]
I have a similar problem, using unicode arrows.

**Comment by howieraem** [comment]
`pip install wcwidth` works for me.

**Comment by themrdjj** [comment]
@howieraem Interesting, for me that doesn't help. I assume you're using the newest versions of both tabulate and wcwidth? In Pycharm, my example still produces the misaligned borders whereas running it in Gnome terminal at least produces correct borders, but the characters are still misaligned.

**Comment by howieraem** [comment]
> @howieraem Interesting, for me that doesn't help. I assume you're using the newest versions of both tabulate and wcwidth? In Pycharm, my example still produces the misaligned borders whereas running it in Gnome terminal at least produces correct borders, but the characters are still misaligned.

I don't know about cases of other languages or symbols, but wcwidth does help it align Chinese characters. I guess you might consider alternatives like `pandas.DataFrame.to_html()`.

**Comment by themrdjj** [comment]
So the fix to this should then be made in https://github.com/jquast/wcwidth?

**Comment by steezynuts** [comment]
I just installed `wcwidth` but it has not fixed this issue for me. I thought that maybe I am just misunderstanding something so apologies if this is unrelated to tabulate/wcwidth.

cat-ing the file via cli returns:
![image](https://user-images.githubusercontent.com/28989869/94200098-75dde700-fe7f-11ea-8c67-bc1069adcd2d.png)

Pasting the value in this comment returns the same issue it returns when the table is emailed to myself.
+-----------------+------------+
| Date            | 2020-09-16 |
| Closing value   | 3385.49    |
| Notional change | -15.71     |
| Percent change  | -0.46      |
+-----------------+------------+

**Comment by astanin** [comment]
This problem is fixed in 0.8.8.
